src/cf-forms.less

```diff
@@ -56,15 +60,32 @@
         <label class="form-label-header">
             Form label header
         </label>
+    - name: Form label helper text
+      markup: |
+        <label class="form-label-helper-text">
+            Form label helper text
+        </label>
   tags:
     - cf-forms
 */
 
 .form-label-header {
-    .h5();
+    .h4();
     margin-bottom: unit(10px / @font-size, em);
 }
 
+.form-label-helper-text {
+    .micro-copy();
+    color: @label-helper;
+    margin-bottom: unit(10px / 14px, em);
```

Comment:
Perhaps note in a comment that the `14px` comes from the font size specified in `.micro-copy()`.